smoke-test/spark-smoke-test/test-spark-lineage/gradlew

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
```

Comment:
why do we need to add in a new gradle here? can we not use the module-based way of building this?